### ISSUE TITLE
Incorrect redirect to 10.10.0.1 when accessing from LAN by dynamic DNS name

### ISSUE THREAD

**Issue by TheSomeMan** [ruuvi/ruuvi.gateway_esp.c#585]

**Comment by markoaamunkajo** [comment]
Could you help with this one @ojousima ?

**Comment by ojousima** [comment]
Dynamic DNS doesn't work from outside LAN right now. 

Here's my setup and result:
1) I configured my DNS ojousima.net to have subdomain sibula.ojousima.net named after my city district.
2) I made a script in my development PC that detects it's IPv4 IP and updates sibula.ojousima.net through Digital Ocean API to point to the local IP. 
3) I configured my home router to forward port 80 from WAN to LAN IP 192.168.1.235, and made the local IP static. Gateway is in that IP.
4) I asked my ISP to open inbound connections to port 80 to my home Internet.
5) I browsed to sibula.ojousima.net with my mobile data connection (outside home LAN)
6) I got redirected to 192.168.1.235/auth.html which does not connect to anything in my mobile data
